src/lib/components/composites/input/ToggleableInput.svelte

```diff
@@ -30,16 +31,27 @@ Usage:
     <ToggleableInput {isEditable} {key} onInputChange={(input) => foo(input)} />
 ```
 -->
-<textarea
-    class={cn(
-        "bg-background text-default min-h-8 w-full resize-none border px-1.5 py-1 focus-visible:outline-hidden",
-        isEditable ? "border-input rounded-md" : "border-transparent",
-        className,
-    )}
-    data-testid={`toggleable-input-${key}`}
-    oninput={(event) => onInputChange((event.target as HTMLTextAreaElement)?.value as string)}
-    {placeholder}
-    readonly={!isEditable}
-    rows="1"
-    {value}
-></textarea>
+{#if isEditable}
+    <Input
+        class="bg-background text-default w-full"
+        data-testid={`toggleable-input-${key}`}
+        inputClass={cn("border-input rounded-md border px-1.5", className)}
+        inputId={`toggleable-input-${key}`}
+        oninput={(event) => onInputChange((event.target as HTMLInputElement)?.value as string)}
+        placeholder={placeholder ?? ""}
+        required
+        type="text"
+        {value}
+    />
+{:else}
+    <div
+        class={cn(
+            "bg-background text-default flex h-10 w-full items-center overflow-hidden px-2",
```

Comment:
```suggestion
            "bg-background text-default flex h-10 w-full items-center overflow-hidden px-1.5",
```
Or why it should be different than in the input?

Reply:
Because the input also has a border. If you set it to 1.5 and toggle the edit mode, you will see the elements move. However, they were a bit off, and I corrected it.

Reply:
I think they are still moving a little bit, that are details that can be ignored in my opinion.